uint16 sensors_data0
uint16 sensors_data1
uint16 sensors_data2
string PS2
string PS3
string PS5
string PS4
string D1D
string D1G
string CP1
string CPI1
string CPI2
string D2D
string D2G
string CP2
string PS6
string PS20
string PS21
string PS22
string PS24
string PS23
string D11D
string D11G
string CP9
string CPI7
string CPI8
string D12D
string D12G
string CP10
string PS1